_posts/2021-06-16-android-flexible-configuration-with-deferred-resources.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Android: Flexible configuration with Deferred Resources"
+excerpt: Deferred Resources provides flexibility beyond the standard resource-resolution approaches.
+tags: Mobile Android Open-source Deferred-Resources
+authors:
+- Drew Hamilton
+header:
+  teaser: TODO
```

Comment:
Not sure what image to use; any suggestions?

Reply:
A simple code snippet using Deferred Resources, made with https://carbon.now.sh/?

Reply:
I used a code snippet, but Carbon's syntax highlighting for Kotlin wasn't very good so I used a screenshot of my IDE instead.